src/rules/functionality.ts

```diff
@@ -6,6 +6,8 @@
 import consoleMethods from './rule-helpers/console-methods';
 
 export default {
+  // Warns for an awaited value that is not a Promise.
+  'await-promise': false,
```

Comment:
For everything marked with feedback, can you run this config through `neutron` and comment with the impact of the rule, please?

Come see me if you need help with that.